scifight_proj/settings.py

```diff
@@ -95,13 +95,18 @@
 
 ADMIN_REORDER = ('sites',
     dict(app='scifight',
+         label='Tournaments',
+         models=[
+             'scifight.Tournament']),
+    dict(app='scifight',
          label='Tournament preparation',
          models=[
              'scifight.Team',
              'scifight.Participant',
              'scifight.Leader',
              'scifight.Room',
              'scifight.Problem',
+             'scifight.Jury',
```

Comment:
That line is unrelated to introducing `Tournament` model, and should live in a separate commit.